packages/core-modules/contracts/mixins/AssociatedSystemsMixin.sol

```diff
@@ -14,6 +14,10 @@ contract AssociatedSystemsMixin is AssociatedSystemsStorage {
     bytes32 internal constant _KIND_ERC721 = "erc721";
     bytes32 internal constant _KIND_UNMANAGED = "unmanaged";
 
+    function _getSystemAddress(bytes32 id) internal view returns (address) {
```

Comment:
Maybe `_getAssociatedSystemAddress` would be a better name (more verbose, but more communicative)